config/forms/2022_2023.json

```diff
@@ -8437,16 +8429,7 @@
                   }
                 ]
               },
-              "informative_text": {
-                "translation": "soft_validations.rent.max.hint_text",
-                "arguments": [
-                  {
-                    "key": "soft_max_for_period",
-                    "label": false,
-                    "i18n_template": "soft_max_for_period"
-                  }
-                ]
-              },
+              "informative_text": {},
```

Comment:
I know we spoke about this, but why did we decide to put it in the hint_text vs informative_text?

Reply:
Ah, nevermind, I just looked at the designs